### PR TITLE
Service Details load Error when Jaeger is configured wrongly

### PR DESCRIPTION
**Pull request by aljesusg**
Fix https://github.com/kiali/kiali/issues/2155

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -130,7 +130,8 @@ type GrafanaVariablesConfig struct {
 // TracingConfig describes configuration used for tracing links
 type TracingConfig struct {
 	Auth              Auth   `yaml:"auth"`
-	Enabled           bool   `yaml:"enabled"` // Enable autodiscover and Jaeger in Kiali
+	Enabled           bool   `yaml:"enabled"` // Enable Jaeger in Kiali
+	Integration       bool   `yaml:",omitempty"`
```
Comment (lucasponce) [inline config/config.go]: `yaml:",omitempty"` > "`yaml:"integration,omitempty"` ?

Note that config changes should be propagated to the CRs

Comment (aljesusg) [reply config/config.go]: This is a prop for use internally with kiali, I think that we shouldn't add to the CRs

Comment (jotak) [reply config/config.go]: if so maybe that shouldn't be in config, but just some global variable?

Comment (aljesusg) [reply config/config.go]: I moved it in the last commit. Thanks !!!

**Comment by aljesusg** [comment]
I'll need to make some changes in the UI too.

**Pull request merged**